rfc002-procope-blog.md

```diff
@@ -0,0 +1,208 @@
+
+*Aside:* We understand there's a blog component to the GDP website. We think that may be a good place to explore hosting white papers, to keep white papers directly attached to the material they support.
+
+Regarding external housing: It's the responsibility of the publishers to make the Terms of Service of those services known to the Project Steering Committee. That includes any changes we're aware of.
+
+* [Medium's Terms of Service](https://policy.medium.com/medium-terms-of-service-9db0094a1e0f) as of 9/12/2020
+* [YouTube's Community Guidelines](https://www.youtube.com/howyoutubeworks/policies/community-guidelines/) help center
+
+### Organization & personnel
+
+First, Procope ideally operates on a "fail early, learn, keep on keeping on." Keep that in mind.
+
+Publication standards are important to all organizations, especially to those promoting publication standards themselves. To that end, there's a hierarchical model borrowed from Ryan's periodical background.
+
+* There's a central **publisher** role, held by two people. They have publication rights on the external accounts (Medium, YouTube). They maintain the publication queue and quality, and have as part of their role to also recruit contributors.
+	* We want to encourage publisher rotation. This could mean one publisher steps down around the time of PSC transitions (around March) or six months afterward, allowing an interested editor to fill that role. This means part of being an editor is getting the specific experience needed to take on the role of publisher. (The first handoff may be bumpy, so we'll play this by ear.)
+	* First publishers would be: Ryan Macklin & Carrie Crowe.
+	* The GDP chair is also in this role, with respect to account access.
+* There are **editors**, who edit pieces to align with our community standards and style. We'll want two editors—and only two editors—involved in any one piece, to keep editors from burning out and to keep the signal/noise ratio down.
+	* Publishers are also editors, and it should go without saying that an author is never their own editor.
+* Anyone in the Procope group can be a **reviewer** when reviews are solicited. Pieces should have reviews solicited.
+* Anyone in the Procope group can submit a proposal for a piece they want to create. Such pieces would be one-off blog posts or videos, and when green lit will go through the process of drafting, reviews, edits, and finally entering the publication queue.
+	* A few authors may do a series, but that will be limited—partly because our publication queue can't maintain several series, and partly because we want to see what contributors provide to the group before promising to support a series.
+* Anyone in the Procope group can add to a topic wishlist.
+
+### Publication rights & ownership
```

Comment:
thought (blocking): I have reservations about supporting proprietary IP publishing, and would like to see this discuss further before locking in. While acknowledging and agreeing with the goals stated, I'm concerned that the encouragement of IP lock-ins may introduce more harm than good. Off-the-top-of-my-head concerns so far:
* Contributors will often be employees of a larger company, and it won't be the individual, but rather the larger company who will claim copyright ownership, and may have legal departments who will aggressively pursue legal avenues. It these situations, you typically are protecting the contributor from their employer. Big company likes the exposure of their employees publishing, so will support the employee writing, and will allow publishing the idea under an open license.
* I'm concerned about "idea squatting". Someone publishing an idea in our forum, then we can't take that idea into our open source templates.
* Or the more likely scenario of "I'm not sure if I can build on someone's idea" and my legal department is very conservative and will just say "NO".
* In almost all cases of great ideas I've seen, the idea is built upon many ideas, refined through many people. Any idea which is only created by one person is both rare, and typically not as refined as we would want. (We want best practices based on many reviewers testing all the edge cases.)
* It is really hard to distill all the nuanced scenarios into a simple legal license - and once the license is in place, it can be gamed and abused. I think it is better to appeal to people's morals. "We expect you to be Good Docs Citizen. If you  build upon someone else's ideas, you should credit and acknowledge them." 
* We are an open source project, and I feel we should honor the ethos of open source in all our material. People attracted to our project will be comfortable with, and I'd suggest would expect that material we generate is under and open license - typically one of the Creative Commons variants.

Reply:
I agree with @camerons that "We are an open source project, and I feel we should honor the ethos of open source in all our material."
It might get confusing for templateers to remember which bits of our project are open and which are not. I'd rather err on the side of everything being open. Anyone who chooses to publish their idea on our forum will know that going in.

Reply:
We could automatically add something like this to the end of every blog post. "<Post Title> © 2021 by <Author Name> is licensed under CC BY 4.0" We could also adopt a DCO type of sign off for commits to the repo, instead of something like a CLA. https://developercertificate.org/

Reply:
Here's where I show my ignorance about open source projects as a contributor: are there projects doing something like this, where there's a personally attributable periodical component alongside a code-based product with docs? I'd love to know how they're doing that.

My model on this is Write the Docs, where the org is itself an open source group, but the talks given at conferences and meetups aren't covered in that same license. While this isn't precisely the same, it's the closest model I have—especially if part of my goal is reaching out to various voices and ask them to guest or more regularly contribute to our more conversational knowledge base. This is part of that "differentiator" element Cameron asked about last week, where I see part of the differentiator as being diverse voices.

What if we took Bryan's suggestion of CC-BY-4.0 as our default rights line, and if someone needs that rights line differing for some reason, that can be a conversation on a case-by-case basis (but needs some justification and isn't a guarantee)?

Reply:
The repository (code and content) are currently under the [Creative Commons Attribution Share Alike 4.0 International](https://github.com/thegooddocsproject/website-hugo/blob/main/LICENSE) which does have a requirement that the same license would be used for a derivative work.

If we use a DCO, then by 'signing' your commit, you would be accepting the same license as the project/repo for your commit/content.   We could also go with a CLA and use something like this to make the process easier. https://cla-assistant.io/

Reply:
Would that DCO model work if, say, Carrie or I commit an article on behalf of an author because we're the leads in the editorial process? Workflow-wise, we don't plan on comitting articles until they've been through editorial. (Google Docs lets is a better commenting/change tracking experience for documents than GitHub is, since we can get to the word and character level rather than just the line level.)

Reply:
I think that DCO clauses (b) and (c) cover the use case where you are submitting content on behalf of someone else.

Reply:
This project might help generate a CLA that is good for our projects in general. http://www.harmonyagreements.org/about.html

Reply:
I would advise strongly for CC-BY on general principles. If a would-be contributor doesn't want to contribute under this license, they could still publish elsewhere and we could include a link to their work in a links roundup. :) 

I'm not very concerned about 'idea squatting' -- short of patent protection and trademark dress (logos, etc.) there is very little protection for ideas as intellectual property. Even if I were the first to publish a blog post on "Blimp-driven documentation: dirigibles as the key channel for developer-focused content" there is nothing to prevent anyone from starting their own line of documentation airships if I haven't patented a specific process or trademarked a term like "DocuBlimp". 😂

Reply:
Generally, I'm pro CC-BY, especially with everything else. I do want to make sure the difference between copyright and license aren't conflated: there isn't a Good Docs Project legal entity to posess copyright, and it's the rights holder that dictates the license (though "dictates" in a publishing platform like this means "we require that as a publication thing).

Noting copyright also semi-covers us in the case of something like plagurism (incl. not crediting other CC-BY licenses), where while we'd have some responsibility as those hosting the content, we don't possess full responsibility because we aren't also the rightsholder.

I don't fear idea squatting, though I respect it and thing we could add a comment about that in our contributor guide. Pretty much for what Erin said, there's technical protection that's functionally non-existent—you've gotta chase down others if they infringe on your copyright. Which sadly enables bad actors (and good actors unintentionally acting poorly due to misinformation), and you can't really stop that, so this in general should be able respect to both the contributor of a piece that involves their voice, and to the community who wants to appreciate the content.

Does that make sense?

Reply:
(For other cases, I've been CC-NC-BY to declare I want people to contact me before monetiing my free content, but that's a different blog model.)

Reply:
@ryanmacklin great point about not conflating copyright and license.

In every case (and I am not a lawyer, just a nerd) unless the author assigns copyright (which is really only done in work-for-hire, and is usually explicit and requires signing something), the copyright belongs to the author. Some people may have work assignments through their day job that would require them to have the copyright assigned back to them (Google has a pretty simple process for this) from their employer.

The *license*, in this case CC-BY, is determined by the author, and the CC-BY license is what gives the GDP permission to publish the content on the blog.

The GDP would only, possibly (and this is where things get hazy) have a copyright on the "anthology" work (that is, the entire blog considered as a whole singular work). This seems unlikely to be an issue.

Now if we're not concerned about anthology rights (and I really think that's not an issue) we could be more liberal in allowing authors to submit under any CC-BY license: CC-BY-NC/ND and possibly SA. I'm having a hard time envisioning a scenario where the GDP would be constrained by a NC/ND license; SA would require that the entire blog be SA, which is also probably not an issue? In that case, anyone republishing any part of the blog would have to make whatever they published also CC-BY-SA. 

Keep in mind that once a work is CC-licensed, that license cannot be changed or revoked. 

Of course, the other thing to consider is that enforcement is pretty thin on the ground. :( I don't think anyone in the GDP will have the time or resources to find bad actors who might republish GDP blog posts on other platforms in license-breaking ways.

Reply:
Re enforcement: the responsibility of that is on the rights holder, which wouldn't be us. (Part of why we likely never want to get into paying for content and making that messy.)

I wrote up a sketch of a contributor guide today, which includes a justification for CC-BY-NC or SA. It's linked on the slack channel, but for ease: https://docs.google.com/document/d/1PZmaNf58-5ZRqPuqlqGjBG4BJok8-nfZpmKZrQ81HE4/edit